broker-cli/commands/health-check.js

```diff
@@ -26,6 +26,92 @@ const ENGINE_STATUS_CODES = Object.freeze({
   VALIDATED: 'VALIDATED'
 })
 
+/**
+ * Prints health check status summary in Table format.
+ * @param {Object} engineStatus
+ * @param {Object} orderbookStatus
+ * @param {Object} relayerStatus
+
+ * @returns {Void}
+ */
+
+function createHealthCheckTable (engineStatus, orderbookStatus, relayerStatus) {
+  const healthcheckTable = new Table({
+    head: ['Component', 'Status'],
+    style: { head: ['gray'] }
+  })
+
+  const ui = []
+
+  ui.push('')
+  ui.push('Sparkswap Healthcheck'.bold.white)
+  ui.push('')
+
+  if (engineStatus.length > 0) {
+    engineStatus.forEach(({ symbol, status }) => {
+      const statusString = (status === ENGINE_STATUS_CODES.VALIDATED) ? `${STATUS_CODES.OK}`.green : status.red
+      healthcheckTable.push([`${symbol} Engine`, statusString])
+    })
+  } else {
+    healthcheckTable.push(['Engines', 'No Statuses Returned'.red])
+  }
+
+  const relayerStatusString = (relayerStatus === STATUS_CODES.OK) ? relayerStatus.green : relayerStatus.red
+  healthcheckTable.push(['Relayer', relayerStatusString])
+
+  if (orderbookStatus.length > 0) {
+    orderbookStatus.forEach(({ market, status }) => {
+      const orderbookStatusString = (status === STATUS_CODES.OK) ? status.green : status.red
+      healthcheckTable.push([`${market} Orderbook`, orderbookStatusString])
+    })
+  } else {
+    healthcheckTable.push(['Orderbooks', 'No Statuses Returned'.red])
+  }
+
+  healthcheckTable.push(['Daemon', `${STATUS_CODES.OK}`.green])
+
+  ui.push(healthcheckTable.toString())
+  console.log(ui.join('\n') + '\n')
+}
+
+/**
+ * Prints health check status summary in JSON format.
+ * @param {Object} engineStatus
+ * @param {Object} orderbookStatus
+ * @param {Object} relayerStatus
+
+ * @returns {Void}
+ */
+
+function createHealthCheckJson (engineStatus, orderbookStatus, relayerStatus) {
+  const statuses = []
+
+  if (engineStatus.length > 0) {
+    engineStatus.forEach(({ symbol, status }) => {
+      const statusString = (status === ENGINE_STATUS_CODES.VALIDATED) ? `${STATUS_CODES.OK}`.green : status.red
```

Comment:
we shouldn't be colorizing output that is used as JSON as colors will mess with the output text and potentially make it unusable for scripting